resharper/resharper-unity/src/Yaml/Psi/Search/UnityYamlUsageSearchFactory.cs

```diff
@@ -45,28 +63,7 @@ public override IEnumerable<string> GetAllPossibleWordsInFile(IDeclaredElement e
         {
             if (IsInterestingElement(element))
             {
-                var words = new List<string>();
-
-                // If it's a class, we only need the asset GUID
-                if (element is IClass)
-                {
-                    var metaFileGuidCache = element.GetSolution().GetComponent<MetaFileGuidCache>();
-                    foreach (var sourceFile in element.GetSourceFiles())
-                    {
-                        // If the element doesn't have the same name as the file it's in, Unity doesn't recognise it
-                        if (!sourceFile.Name.StartsWith(element.ShortName))
-                            continue;
-
-                        var guid = metaFileGuidCache.GetAssetGuid(sourceFile);
-                        if (guid != null)
-                            words.Add(guid);
-                    } 
-                }
-                else
-                {
-                    words.Add(element.ShortName);
-                }
-
+                var words = new List<string> {YamlTrigramIndexBuilder.YAML_REFERENCE_IDENTIFIER};
```

Comment:
Won't this include every YAML asset file? Should there be more words like the GUID, etc. to narrow it down?

Reply:
No, because adding names will require to construct correct trigram index. Correct trigram index will require reading document at start-up

Reply:
No, because adding names will require to construct correct trigram index. Correct trigram index will require reading document at start-up